.gitignore

```diff
@@ -0,0 +1,2 @@
+cypress/
+node_modules/
```

Comment:
???

Reply:
The cypress folder is generated on test run

Reply:
You don't have a new line at the end of this file